test/Microsoft.TestPlatform.AcceptanceTests/CodeCoverageTests.cs

```diff
@@ -131,7 +268,31 @@ private static void AssertSourceFileName(CoverageDS coverageDS)
                 $"Code Coverage not collected for file: {expectedFileName}");
         }
 
-        private void AssertModuleCoverageCollected(CoverageDS coverageDS)
+        private void AssertSkippedMethod(CoverageDS coverageDS)
+        {
+            var dict = new Dictionary<string, bool>()
+            {
+                {"TestSign()", false},
+                {"TestAbs()", true}
+            };
+
+            for (int i = 0; i < coverageDS.Method.Count; ++i)
+            {
```

Comment:
theoretically the reason that function is not covered can be that we failed instrumentation. So we should check it better.

Inside coverage report you will find such thing:

      <skipped_functions>
        <skipped_function id="8272" token="0x6000001" name="Resources()" type_name="Resources" reason="attribute_excluded" />
        <skipped_function id="8284" token="0x6000002" name="get_ResourceManager()" type_name="Resources" reason="attribute_excluded" />
        <skipped_function id="8356" token="0x6000003" name="get_Culture()" type_name="Resources" reason="attribute_excluded" />
        <skipped_function id="8379" token="0x6000004" name="set_Culture(System.Globalization.CultureInfo)" type_name="Resources" reason="attribute_excluded" />
        <skipped_function id="8388" token="0x6000005" name="get_AbortedMultiTestRunFinalization()" type_name="Resources" reason="attribute_excluded" />
        <skipped_function id="8428" token="0x6000006" name="get_AbortedTestsDiscovery()" type_name="Resources" reason="attribute_excluded" />
        <skipped_function id="8468" token="0x6000007" name="get_AbortedTestsRun()" type_name="Resources" reason="attribute_excluded" />
        <skipped_function id="8508" token="0x6000008" name="get_FailedToReceiveMessage()" type_name="Resources" reason="attribute_excluded" />
        <skipped_function id="8548" token="0x6000009" name="get_InvalidFilePath()" type_name="Resources" reason="attribute_excluded" />
        <skipped_function id="8588" token="0x600000a" name="get_VsTestProcessExitedAbnormally()" type_name="Resources" reason="attribute_excluded" />
      </skipped_functions>

you should verify reason here

Reply:
Done.